web/src/main/java/org/cbioportal/web/StudyController.java

```diff
@@ -68,6 +95,17 @@
         @RequestParam(required = false) StudySortBy sortBy,
         @ApiParam("Direction of the sort")
         @RequestParam(defaultValue = "ASC") Direction direction) {
+        
+        if (!usingAuth()
```

Comment:
In a password protected portal, its probably true that a complete set of studies is different between registered users.  The use of !usingAuth ensures that defaultResponse is only used in an unprotected portal when the defaultResponse is the same for all users.